packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -180,8 +211,9 @@ class TextPainter {
         textAlign: textAlign,
         textScaleFactor: textScaleFactor,
         ellipsis: _ellipsis,
+        maxLines: _maxLines,
       );
-      paragraphStyle ??= new ui.ParagraphStyle();
+      paragraphStyle ??= new ui.ParagraphStyle(maxLines: _maxLines);
```

Comment:
I don't understand why `maxLines: _maxLines` is here but the other properties, like `ellipsis` aren't here.